pkg/controller/controller_binding.go

```diff
@@ -1333,10 +1376,10 @@ func (c *controller) processUnbindAsyncResponse(binding *v1beta1.ServiceBinding,
 // an error returned during reconciliation while polling a service binding.
 func (c *controller) handleServiceBindingPollingError(binding *v1beta1.ServiceBinding, err error) error {
 	// During polling, an error means we should:
-	//	1) log the error
-	//	2) attempt to requeue in the polling queue
-	//		- if successful, we can return nil to avoid regular queue
-	//		- if failure, return err to fall back to regular queue
+	// 	1) log the error
```

Comment:
what are these new spaces about?

Reply:
Looks like my IDE thinks an extra space is needed here :).  I've fixed it now.